internal/pkg/deploy/cloudformation/stack/worker_svc_test.go

```diff
@@ -165,7 +165,6 @@ Outputs:
 						Network: template.NetworkOpts{
 							AssignPublicIP: template.DisablePublicIP,
 							SubnetsType:    template.PrivateSubnetsPlacement,
-							SecurityGroups: []string{"sg-1234"},
```

Comment:
Why are we removing this from all the tests?

Reply:
I think stringOrFromCFN{} can not be exported from manifest package. So that is the reason for removing this test case.